src/v/pandaproxy/schema_registry/sharded_store.cc

```diff
@@ -348,16 +362,34 @@ ss::future<std::vector<schema_id>> sharded_store::referenced_by(
         ver = versions.back();
     }
 
-    auto map = [sub{std::move(sub)}, ver](store& s) {
-        return s.referenced_by(sub, ver);
-    };
-    auto reduce = [](std::vector<schema_id> acc, std::vector<schema_id> refs) {
-        acc.insert(acc.end(), refs.begin(), refs.end());
-        return acc;
-    };
+    // Find all the schema that reference this sub-ver
     auto references = co_await _store.map_reduce0(
-      map, std::vector<schema_id>{}, reduce);
-    std::sort(references.begin(), references.end());
+      [sub{std::move(sub)}, ver](store& s) {
+          return s.referenced_by(sub, ver);
+      },
+      std::vector<schema_id>{},
+      [](std::vector<schema_id> acc, std::vector<schema_id> refs) {
+          acc.insert(acc.end(), refs.begin(), refs.end());
+          return acc;
+      });
+    absl::c_sort(references);
```

Comment:
is htis different than std::sort used above?

Reply:
Fixed by using `absl::btree_set`